src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -66,12 +66,12 @@ public static async Task<RemoteHostClient> CreateAsync(Workspace workspace, bool
             Stream stream)
             : base(workspace)
         {
-            Contract.ThrowIfNull(remotableDataRpc);
+            Contract.ThrowIfTrue(remotableDataRpc.IsDefault);
 
             ClientId = clientId;
 
             _inprocServices = inprocServices;
-            _remotableDataRpc = remotableDataRpc;
+            _remotableDataRpc = remotableDataRpc.Move();
```

Comment:
📝 Most of the calls to `Move` can be eliminated by using flow analysis in the non-copyable analyzer and relying on the analyzer to detect errors. The main advantage of the `Move` method is it leverages the `[MaybeNull]` post-condition attribute to trigger compile time detection of coding errors using the proposed non-defaultable value types support.